src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/SlingInitialContentBundleEntryMetaDataCollector.java

```diff
@@ -138,6 +141,11 @@ private void extractFile(JarEntry jarEntry, JarOutputStream bundleOutput) throws
 
                 SlingInitialContentBundleEntryMetaData bundleEntry = createSlingInitialContentBundleEntry(context, targetFile);
                 collectedSlingInitialContentBundleEntries.add(bundleEntry);
+            } else if (jarEntry.getName().endsWith(".cnd")) {
```

Comment:
IMHO only those cnd files referenced in the manifest are considered in Sling Initial Content (https://sling.apache.org/documentation/bundles/content-loading-jcr-contentloader.html#node-type-and-namespace-definitions-cnd)

Reply:
Good point. Let me work on that.

Reply:
@kwin  implemented & pushed